src/commands/compress.rs

```diff
@@ -45,15 +46,16 @@ pub fn compress_files(
     // Grab previous encoder and wrap it inside of a new one
     let chain_writer_encoder = |format: &_, encoder| -> crate::Result<_> {
         let encoder: Box<dyn Send + Write> = match format {
-            Gzip => Box::new(
+            Gzip => Box::new({
                 // by default, ParCompress uses a default compression level of 3
                 // instead of the regular default that flate2 uses
-                gzp::par::compress::ParCompress::<gzp::deflate::Gzip>::builder()
```

Comment:
Interface updated to now include a new `W` generic. Just using `gzp::par::compress::ParCompress::<gzp::deflate::Gzip, _>::builder()` didn't work, unless I'm brainfarting.

Using `ParCompressBuilder::new()` as per the [gzp examples](https://github.com/sstadick/gzp/blob/main/examples/test1.rs) worked fine, though